libraries/Wire/Wire.cpp

```diff
@@ -47,9 +44,9 @@ void TwoWire::_begin() {
 
     if (!is_master) {
         // Configure the read and write buffers for the I2C slave
-        w_status = cyhal_i2c_slave_config_read_buffer(&i2c_obj,  txBuffer, BUFFER_LENGTH);
+        w_status = cyhal_i2c_slave_config_read_buffer(&i2c_obj,  txBuffer._aucBuffer, BUFFER_LENGTH);
         Wire_assert(w_status);
-        w_status = cyhal_i2c_slave_config_write_buffer(&i2c_obj, rxBuffer, BUFFER_LENGTH);
+        w_status = cyhal_i2c_slave_config_write_buffer(&i2c_obj, rxBuffer._aucBuffer, BUFFER_LENGTH);
```

Comment:
And here? The master writes always in the first position of the circular buffer. Then the slaves reads from the circular buffer? 
I am not sure if this would work...